src/Compatibility/Core/src/Compatibility.csproj

```diff
@@ -1,17 +1,26 @@
-﻿<Project Sdk="MSBuild.Sdk.Extras">
+﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0;$(MauiPlatforms)</TargetFrameworks>
+    <TargetFrameworks>netstandard2.1;netstandard2.0;$(_MauiDotNetTfm);$(MauiPlatforms)</TargetFrameworks>
     <RootNamespace>Microsoft.Maui.Controls.Compatibility</RootNamespace>
     <AssemblyName>Microsoft.Maui.Controls.Compatibility</AssemblyName>
     <Nullable>disable</Nullable>
     <AndroidRoot>Android\</AndroidRoot>
     <iOSRoot>iOS\</iOSRoot>
     <WindowsRoot>Windows\</WindowsRoot>
+    <IsPackable>false</IsPackable>
   </PropertyGroup>
-  <PropertyGroup>
-    <PackageId>Microsoft.Maui.Controls.Compatibility</PackageId>
+
+  <PropertyGroup Condition="$(TargetFramework.Contains('-windows')) == true ">
```

Comment:
This project got some auto changes that need to be reviewed and probably some reverted.

Reply:
Ok, these are actually ok, the diff is simply weird as the -net6 was dropped and looks in the diff as deleted so the diff compares against the legacy project as @PureWeen pointed out.